### PR TITLE
notifications: refactor to remove Configuration global variable

### PR DESCRIPTION
**Pull request by michaelestrin**
Fixes: #2062

Signed-off-by: Michael Estrin m.estrin@dell.com

### CODE REVIEW

**Comment by michaelestrin** [comment]
Builds on https://github.com/edgexfoundry/edgex-go/pull/2065

**Comment by michaelestrin** [comment]
Rebased.

File: internal/support/notifications/container/config.go [hunk 1]
```diff
@@ -1,7 +1,5 @@
 /*******************************************************************************
- * Copyright 2017 Dell Inc.
- * Copyright 2018 Dell Technologies Inc.
- * Copyright (c) 2019 Intel Corporation
```
Comment (brandonforster) [inline internal/support/notifications/container/config.go]: According to the Apache License, derivative works must have original copyrights preserved. Please restore the originals.

Comment (michaelestrin) [reply internal/support/notifications/container/config.go]: This change was made by Goland when I moved the file from one directory to another.  It wasn't an intentional change on my part and I missed it when I reviewed the changes prior to creating a PR.

While I don't agree that their remove is a violation of the Apache license, I will manually restore the copyright chain as requested.

**Comment by michaelestrin** [comment]
Restored copyright notices as requested.

File: internal/support/notifications/container/config.go (hunk 1 above)
Comment (brandonforster) [reply internal/support/notifications/container/config.go]: Totally understand. I just don't want to do anything that might offend any companies or come ANYWHERE NEAR lawyers getting mad.

**Pull request merged**